example/Clarinet.toml

```diff
@@ -26,8 +34,8 @@ path = "contracts/slice.clar"
 clarity_version = 3
 epoch = 3.0
 
-[contracts.trait]
-path = 'contracts/trait.clar'
+[contracts.ft-transfer-many]
```

Comment:
Perhaps a more friendly name could be `send-tokens`? We have `counter`, `cargo`, etc. (Like, imagine if `cargo` was called `creade-read-upate-delete.clar`.) 👀